tests/test_group.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from random import randrange
+
+import pytest
+import time
+
+from selenium.webdriver.common.by import By
+from pages.home_page import Home
+
+
+class TestGroup:
+
+    @pytest.mark.credentials
+    def test_group_description_edit(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        # New group data
+        new_group_description_text = 'This is an automated group.'
+        new_group_irc_channel = '#testgroup'
+
+        group_description = group.description.description_info
+
+        # Update the group description fields
+        group_description.set_description_text(new_group_description_text)
+        group_description.set_irc_channel(new_group_irc_channel)
+        group_description.click_update()
+
+        search_listings = home_page.header.search_for(group_name)
+        assert search_listings.is_element_present(By.LINK_TEXT, group_name)
```

Comment:
We should avoid locators in tests. The next line would fail if the group was not present, so I think we can remove this line.